lib/arc.d.ts

```diff
@@ -234,6 +234,10 @@ declare module "daostack-arc-js" {
     // vote(proposalId, choice, params);
     static new(options: OrganizationNewConfig): Promise<Organization>;
     static at(avatarAddress: string): Promise<Organization>;
+
+    getName(): string;
+    getTokenName(): string;
```

Comment:
Can you please add comments describing what these three new methods do? (like *what* name is returned, and so on)?  Thanks.

Reply:
sure

Reply:
Heh, should be "Organization name", not "DAO"  Sorry ;-)